src/docs/metrics/holders-distribution/index.md

```diff
@@ -0,0 +1,155 @@
+---
+title: Holders Distribution
+author: Santiment Team
+date: 2020-08-24
+---
+
+## Definition
+
+There are five separate metrics regarding holders distribution:
+
+- holders_distribution_<interval> - represents the number of addresses holding the amount of tokens that belong to a given interval
+- holders_distribution_combined_balance_<interval> - represents the sum of all the tokens of all the addresses which hold the amount of tokens from an interval
+- holders_distribution_over_<threshold> - the number of addresses whose balance exceeds threshold
```

Comment:
exceeds `threshold` so it's clearer that it is replaced by the actual value in the metric name